docs/ml/upload-model.md

```diff
@@ -41,32 +41,42 @@ Select **Next steps** to continue.
 
 {{<imgproc src="/ml/upload-model.png" resize="600x" alt="Upload model menu on the DATA tab of the Viam app.">}}
 
-1. Specify a name for the model.
-2. Specify the **Model type**.
-3. Add a `.tflite` model file.
-4. (Optional) Add a `.txt` label file.
-   This should include the label names as you provided them in training, with one name per line.
+1. Select the **Model framework** or the type of model.
+   TensorFlow Lite, TensorFlow, PyTorch and ONNX model frameworks are currently supported.
+2. Upload the files required for your model framework.
+3. Specify a name for the model.
+4. Specify the **Task type**.
 5. Add a short description.
    This will help other users to understand how to use your model.
 
 Select **Upload model**.
 The model is now visible on the **MODELS** subtab of the **DATA** tab in the [Viam app](https://app.viam.com) and available to [deploy on your machine](/ml/deploy/).
 Also, if you selected **Public**, it should be publicly visible in the [Viam registry](https://app.viam.com/registry).
 
+{{% alert title="TFLite label files" color="tip" %}}
+During Step 2, if you chose a TensorFlow Lite model framework, you add a `.txt` label file.
+This should include the label names as you provided in training, with one name per line.
```

Comment:
```suggestion
This should include the label names you provided in training, with one name per line.
```
Sorry with my first suggestion on this line I missed this part-- I think this makes more sense??